back/db/entrypoint/init.sql

```diff
@@ -11,3 +11,10 @@ CREATE TABLE IF NOT EXISTS news (
     created_date DATE NOT NULL,
     modified_date DATE NOT NULL
 );
+
+CREATE TABLE IF NOT EXISTS users (
+    uid         INT             AUTO_INCREMENT  PRIMARY KEY,
+    email       VARCHAR(64)     NOT NULL        UNIQUE,
```

Comment:
Why the email's field should be `VARCHAR(64)`? Why not `TEXT`?

Reply:
Using TEXT can be complex under the constraint UNIQUE.

Reply:
OK. Get it. https://stackoverflow.com/a/574698/13031497, but you can change its length bigger.